junebug/stores.py

```diff
@@ -1,6 +1,17 @@
 from twisted.internet.defer import inlineCallbacks, returnValue
+from twisted.web import http
+
 from vumi.message import TransportUserMessage
 
+from junebug.error import JunebugError
+
+
+class MessageNotFound(JunebugError):
+    '''Raised when a message is not found.'''
+    name = 'MessageNotFound'
+    description = 'message not found'
+    code = http.NOT_FOUND
```

Comment:
So the action is "reply to this message" and we're saying "that message doesn't exist". So I would say it's a 400 Bad Request, but I think it would be good to have a discussion on this.